README.md

```diff
@@ -48,6 +48,11 @@ To learn more about Next.js, take a look at the following resources:
 
 - [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
 
+## Build Status
```

Comment:
This should be the first thing, add the bade without this ## Build status header

Reply:
@Mide001 please review